### ISSUE TITLE
Remove redundancy in logs

### ISSUE THREAD

**Issue by sjezewski** [pachyderm/pachyderm#987]
Right now we double report the end of the request. (see below)

Let's remove the double logging. This should help our log size by roughly 1/3 (since we still want to report the initial request separately).

`````` shell
2016-10-18T19:16:07Z INFO  protorpclog.Call {"service":"pachyderm.pfsserver.API","method":"ListFile","request":"file:\u003ccommit:\u003crepo:\u003cname:\"test_input\" \u003e id:\"master/32\" \u003e path:\"/115\" \u003e shard:\u003cfile_modulus:1 block_modulus:1 \u003e recurse:true ","response":"file_info:\u003cfile:\u003ccommit:\u003crepo:\u003cname:\"test_input\" \u003e id:\"master/32\" \u003e path:\"/115/capvoltage\" \u003e file_type:FILE_TYPE_DIR size_bytes:868168 modified:\u003cseconds:1476815878 nanos:720432745 \u003e commit_modified:\u003crepo:\u003cname:\"test_input\" \u003e id:\"master/15\" \u003e \u003e file_info:\u003cfile:\u003ccommit:\u003crepo:\u003cname:\"test_input\" \u003e id:\"master/32\" \u003e path:\"/115/digitizer\" \u003e file_type:FILE_TYPE_DIR size_bytes:18199608 modified:\u003cseconds:1476815869 nanos:477991529 \u003e commit_modified:\u003crepo:\u003cname:\"test_input\" \u003e id:\"master/15\" \u003e \u003e file_info:\u003cfile:\u003ccommit:\u003crepo:\u003cname:\"test_input\" \u003e id:\"master/32\" \u003e path:\"/115/magnet\" \u003e file_type:FILE_TYPE_DIR size_bytes:55675 modified:\u003cseconds:1476815870 nanos:47484791 \u003e commit_modified:\u003crepo:\u003cname:\"test_input\" \u003e id:\"master/15\" \u003e \u003e file_info:\u003cfile:\u003ccommit:\u003crepo:\u003cname:\"test_input\" \u003e id:\"master/32\" \u003e path:\"/115/shotlog\" \u003e file_type:FILE_TYPE_DIR size_bytes:841 modified:\u003cseconds:1476815870 nanos:110806928 \u003e commit_modified:\u003crepo:\u003cname:\"test_input\" \u003e id:\"master/15\" \u003e \u003e ","duration":"0.110868506s"}
2016-10-18T19:16:07Z INFO  protorpclog.Call {"service":"pfs.API","method":"ListFile","request":"file:\u003ccommit:\u003crepo:\u003cname:\"test_input\" \u003e id:\"master/32\" \u003e path:\"/115\" \u003e shard:\u003cfile_modulus:1 block_modulus:1 \u003e recurse:true ","response":"file_info:\u003cfile:\u003ccommit:\u003crepo:\u003cname:\"test_input\" \u003e id:\"master/32\" \u003e path:\"/115/capvoltage\" \u003e file_type:FILE_TYPE_DIR size_bytes:868168 modified:\u003cseconds:1476815878 nanos:720432745 \u003e commit_modified:\u003crepo:\u003cname:\"test_input\" \u003e id:\"master/15\" \u003e \u003e file_info:\u003cfile:\u003ccommit:\u003crepo:\u003cname:\"test_input\" \u003e id:\"master/32\" \u003e path:\"/115/digitizer\" \u003e file_type:FILE_TYPE_DIR size_bytes:18199608 modified:\u003cseconds:1476815869 nanos:477991529 \u003e commit_modified:\u003crepo:\u003cname:\"test_input\" \u003e id:\"master/15\" \u003e \u003e file_info:\u003cfile:\u003ccommit:\u003crepo:\u003cname:\"test_input\" \u003e id:\"master/32\" \u003e path:\"/115/magnet\" \u003e file_type:FILE_TYPE_DIR size_bytes:55675 modified:\u003cseconds:1476815870 nanos:47484791 \u003e commit_modified:\u003crepo:\u003cname:\"test_input\" \u003e id:\"master/15\" \u003e \u003e file_info:\u003cfile:\u003ccommit:\u003crepo:\u003cname:\"test_input\" \u003e id:\"master/32\" \u003e path:\"/115/shotlog\" \u003e file_type:FILE_TYPE_DIR size_bytes:841 modified:\u003cseconds:1476815870 nanos:110806928 \u003e commit_modified:\u003crepo:\u003cname:\"test_input\" \u003e id:\"master/15\" \u003e \u003e ","duration":"0.111561143s"}```
``````

**Comment by sjezewski** [comment]
I'm also seeing a lot of lines like:

```
2016-11-01T20:34:20Z INFO  protorpclog.Call {"service":"health.Health","method":"Health","duration":"0.000001291s"}
```

I see about 20 of those per second, and those messages are about 4KB, so that means we have roughly 6GB of logs / day just from this log type

6gb / day = 4KB * 20 messages/s * 60 s/min * 60 min/hr * 24hr/day   /   1024 kb/mb   /   1024 mb/gb

**Comment by jdoliner** [comment]
This has been fixed by #1018.
